sources/Renderer/OpenGL/Platform/Linux/LinuxGLSwapChainContext.cpp

```diff
@@ -22,58 +26,123 @@ namespace LLGL
 
 std::unique_ptr<GLSwapChainContext> GLSwapChainContext::Create(GLContext& context, Surface& surface)
 {
-    return MakeUnique<LinuxGLSwapChainContext>(static_cast<LinuxGLContext&>(context), surface);
+#ifdef LLGL_LINUX_ENABLE_WAYLAND
+    if (surface.IsWayland()) {
```

Comment:
Please use [Allman indentation](https://en.wikipedia.org/wiki/Indentation_style#Notable_styles) style in this project.